pubsub/gcloud/aio/pubsub/metrics.py

```diff
@@ -0,0 +1,41 @@
+from gcloud.aio.auth import BUILD_GCLOUD_REST
+
+if BUILD_GCLOUD_REST:
+    pass
+else:
+    import prometheus_client
+
+    _NAMESPACE = 'gcloud_aio'
+    _SUBSYSTEM = 'pubsub'
+
+    CONSUME = prometheus_client.Counter(
+        'subscriber_consume',
+        'Counter of the outcomes of PubSub message consume attempts',
+        ['outcome'],
+        namespace=_NAMESPACE,
+        subsystem=_SUBSYSTEM,
+        unit='total')
+
+    CONSUME_LATENCY = prometheus_client.Histogram(
+        'subscriber_consume_latency',
+        'Histogram of PubSub message consume latencies',
+        ['aspect'],
+        namespace=_NAMESPACE,
+        subsystem=_SUBSYSTEM,
+        unit='seconds')
+
+    BATCH = prometheus_client.Counter(
+        'subscriber_batch',
```

Comment:
I might argue for giving this a `_status` suffix (eg. since it kind of matches the `job_status` we've been using in other PRs). Alternatively, for removing the `_status` suffix elsewhere. Thoughts? I'm mostly looking for places where we should encode a bit of consistency at this point.

Reply:
I do like batch status better than just `batch`. Something about `batch` was bothering me.